site/installing/cloud.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Installing Weave Flux with Weave Cloud
+menu_order: 10
+---
+
+In addition to visualising and monitoring your cloud native application,
+[Weave Cloud](https://cloud.weave.works) also provides a fully hosted
+version of Flux. The use of weave cloud dramatically simplifies the
+installation and ongoing operation of Flux.
+
+Using Flux on [Weave Cloud](https://cloud.weave.works) requires the
+installation of two components:
+
+-   the command-line client **fluxctl** and
+-   the daemon, **fluxd** which carries out tasks on behalf of the
+hosted service
+
+# Installing the Daemon
+
+Sign up with [Weave Cloud](https://cloud.weave.works) and create an
+instance to represent your cluster. If you're already using Scope or
+Cortex to look at a cluster, you can choose that instance instead of
+creating one.
+
+We need the "service token" for your instance, which is a string of
+random characters generated by Weave Cloud, like
+`dkmbr4spss8r4iz1115xo89pzjfib8h7`. You can get this from the settings
+page for the instance.
+
+Next, install the Flux daemon on your k8s cluster. This allows the 
+Flux service in Weave Cloud to communicate with your cluster.
+
+```sh
+kubectl apply -f 'https://cloud.weave.works/k8s/flux.yaml?service-token=<your service token>'
+```
+
+Where you should replace `<your service token>` with the token 
+described above.
+
+# Installing the Client
+
+Now that the daemon is installed, we need to install a client called 
+fluxctl to interact with the service. Download the
+[fluxctl client from github](https://github.com/weaveworks/flux/releases/latest).
+
+For OSX, you can use this one-liner:
+
+```sh
+curl -L $(curl -s https://api.github
+.com/repos/weaveworks/flux/releases | grep browser_download_url | grep darwin | head -n 1 | cut -d '"' -f 4) -o /usr/local/bin/fluxctl && chmod +x fluxctl
+```
```

Comment:
That looks a bit like it downloads it to `/usr/local/bin' then chmods a file in the working directory, am I reading that wrong?